containers/nginx/minds.conf

```diff
@@ -1,3 +1,12 @@
+# Dynamic language redirection.
+# Ensure that "en" is last, or people with, for instance, "fr en" will always default to "en".
+map $http_accept_language $lang {
+    default en;
+    ~es es;
+    ~fr fr;
```

Comment:
This list will need to be kept in sync with the languages supported by the application. There are methods to automatically sync this, particularly if you run Docker on production.